pwndbg/hexdump.py

```diff
@@ -66,51 +79,110 @@ def load_color_scheme():
+            yield H.offset("+%04x " % len(data)) + H.address("%#08x  " % (address + len(data)))
+
+            # Don't allow iterating over this generator again
+            return
+
+        data = list(bytearray(data))
+        last_line = None
+        skipping = False
+        for i, line in enumerate(groupby(width, data, fill=-1)):
+            if skip and line == last_line:
+                if skipping:
+                    continue
+
+                skipping = True
+                yield "..."
+            else:
+                skipping = False
+                last_line = line
+
+            hexline = []
+
+            hexline.append(H.offset("+%04x " % ((i + offset) * width)))
+            hexline.append(H.address("%#08x  " % (address + (i * width))))
+
+            for group in groupby(group_width, line):
+                group = reversed(group) if flip_group_endianess else group
+                for idx, char in enumerate(group):
+                    if flip_group_endianess and idx == group_width - 1:
+                        hexline.append(H.highlight_group_lsb(color_scheme[char]))
+                    else:
+                        hexline.append(color_scheme[char])
+                    hexline.append(str(config_byte_separator))
+                hexline.append(" ")
 
-            skipping = True
-            yield "..."
-        else:
-            skipping = False
-            last_line = line
-
-        hexline = []
-
-        hexline.append(H.offset("+%04x " % ((i + offset) * width)))
-        hexline.append(H.address("%#08x  " % (address + (i * width))))
-
-        for group in groupby(group_width, line):
-            group = reversed(group) if flip_group_endianess else group
-            for idx, char in enumerate(group):
-                if flip_group_endianess and idx == group_width - 1:
-                    hexline.append(H.highlight_group_lsb(color_scheme[char]))
-                else:
-                    hexline.append(color_scheme[char])
-                hexline.append(str(config_byte_separator))
-            hexline.append(" ")
-
-        hexline.append(H.separator("%s" % config_separator))
-        for group in groupby(group_width, line):
-            for char in group:
-                hexline.append(printable[char])
             hexline.append(H.separator("%s" % config_separator))
+            for group in groupby(group_width, line):
+                for char in group:
+                    hexline.append(printable[char])
+                hexline.append(H.separator("%s" % config_separator))
+
+            yield "".join(hexline)
+
+    else:
+
+        """
+        Traditionally, windbg will display 16 bytes of data per line.
+        """
+        values = []
+
+        if repeat:
+            count = hexdump.last_count
+            address = hexdump.last_address
+        else:
+            address = int(address) & pwndbg.gdblib.arch.ptrmask
+            count = int(count)
+
+        type = get_type(size)
```

Comment:
`type` is a built-in Python function. In the future we'll have a linter preventing names like this, so could you rename the variable to something else? If `type` really is the best name for this, we can go with something like `type_`.